### PR TITLE
Add EDS search bar 

### PR DESCRIPTION
**Pull request by suecarmol**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Added the EBSCO Discovery Service search bar to the header

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
As part of the new redesign, we want to integrate the EBSCO Discovery Service search bar. This should be in the site header and accessible from all pages carrying the header.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T240128](https://phabricator.wikimedia.org/T240128)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/templates/header_partial_b4.html [hunk 1]
```diff
@@ -11,11 +11,34 @@
...
+    <input name="schemaId" value="search" type="hidden" />
+    <input name="custid" value="ns253359" type="hidden" />
+    <input name="groupid" value="main" type="hidden" />
+    <input name="profid" value="eds" type="hidden" />
+    <input name="proxy" value="https://wikipedialibrary.idm.oclc.org:9443/login?url=" type="hidden" />
+    <input name="scope" value="site" type="hidden" />
+    <input name="site" value="eds-live" type="hidden" />
+    <input name="direct" value="true" type="hidden" />
+    <div class="input-group eds-searchbox">
+      {% comment %}Translators: Shown in the EBSCO Discovery Search input. {% endcomment %}
```
Comment (Samwalton9) [inline TWLight/templates/header_partial_b4.html:30]: I'd probably leave "EBSCO Discovery" out of this comment, it's more likely to be confusing to translators I think.

Comment (suecarmol) [reply TWLight/templates/header_partial_b4.html]: Done!

**Comment by Samwalton9** [comment]
Does this incorporate the language mapping we talked about where the following need to be adjusted?

- zh_Hant = zh-tw
- ch_Hans = zh-cn
- pt = pt-pt
- pt_Br = pt-br

**Comment by jsnshrmn** [comment]
:point_up:  good catch!

**Comment by suecarmol** [comment]
> Does this incorporate the language mapping we talked about where the following need to be adjusted?
> 
>     * zh_Hant = zh-tw
> 
>     * ch_Hans = zh-cn
> 
>     * pt = pt-pt
> 
>     * pt_Br = pt-br

Not yet! Will get to it

**Comment by suecarmol** [comment]
@Samwalton9 @jsnshrmn I have added an if block because the mapping is for only 3 languages (`pt-br` already works). Let me know if I need to make further changes

**Review by jsnshrmn — APPROVED** [review]
:+1:  This is a great way to handle this considering the small number of locales to match

**Comment by suecarmol** [comment]
Moving on to merge this on the feature branch

**Pull request merged**